courses.json

```diff
@@ -391,5 +391,22 @@
         ]
       }
     ]
+  },
+  {
+    "id": "Artificial Intelligence",
+    "name": "Artificial Intelligence Master's Degree",
+    "icon": "🧠",
+    "years": [
+      {
+        "year": 1,
+        "teachings": [
+          {
+            "name": "Deep Learning",
+            "url": "deep-learning",
+            "chat": ""
```

Comment:
The chat link is available here https://github.com/csunibo/config/issues/8. Just click on "groupchat links" or whatever in the OP.